app/tests/components/Form/ApplicationFormStatus.test.tsx

```diff
@@ -59,22 +59,23 @@ describe('The application form', () => {
   it('displays the saved date when it was saved on a different day', () => {
     const mockCurrentTime = DateTime.utc(2020, 1, 2, 0, 0);
     Settings.now = () => mockCurrentTime.toMillis();
-    Settings.defaultZone = 'UTC';
+    Settings.defaultZone = "UTC";
+    Settings.defaultLocale = "en-CA";
 
     componentTestingHelper.loadQuery();
-    componentTestingHelper.renderComponent();
+    componentTestingHelper.renderComponent(false);
 
     expect(screen.queryByText('Saving')).toBeNull();
-    expect(screen.getByText('Last saved: Jan. 1')).toBeInTheDocument();
+    expect(screen.getByText('Last saved: Jan 1')).toBeInTheDocument();
   });
 
   it('displays the error message if provided', () => {
     const mockCurrentTime = DateTime.utc(2020, 1, 2, 0, 0);
     Settings.now = () => mockCurrentTime.toMillis();
-    Settings.defaultZone = 'UTC';
+    Settings.defaultZone = "America/Vancouver";
```

Comment:
This line can probably be removed now that `renderComponent` will set the default date and zone